gather.go

```diff
@@ -598,7 +617,8 @@ func (a *Agent) gatherCandidatesRelay(ctx context.Context, urls []*URL) { //noli
 				relayProtocol = "dtls"
 				locConn = &fakePacketConn{conn}
 			case url.Proto == ProtoTypeTCP && url.Scheme == SchemeTypeTURNS:
-				conn, connectErr := tls.Dial(NetworkTypeTCP4.String(), TURNServerAddr, &tls.Config{
+				network = tcpNetworkType
```

Comment:
Previously, `network` was hardcoded to `NetworkTypeUDP4` even if the `url.Proto == ProtoTypeTCP`.
Not sure if this was correct?